config/checkstyle-examples-suppressions.xml

```diff
@@ -62,6 +62,10 @@
   <suppress checks="CommentsIndentation"
     files="commentsindentation[\\/]Example[1-9].java"/>
 
+  <!-- violation required by to show behavior of Check -->
+  <suppress checks="FileLength"
+    files="cyclomaticcomplexity[\\/]Example\d+.*"/>
```

Comment:
Please remove this, your examples below 60 line limit now

Reply:
I think It checks the length of the entire file not just the example so we need the suppression.
<img width="1369" alt="Screenshot 2024-11-20 at 5 16 30 AM" src="https://github.com/user-attachments/assets/f07c4641-577f-4289-bf5d-8d0488323037">

Reply:
Ok, in this case let's keep suppression